packages/codegen-ui-react/lib/helpers/index.ts

```diff
@@ -14,3 +32,16 @@
   limitations under the License.
  */
 export const lowerCaseFirst = (input: string) => input.charAt(0).toLowerCase() + input.slice(1);
+
+export const addModelAlias = (componentTypes: string[], importCollection: ImportCollection, model: string) => {
+  if (componentTypes.includes(model) || isPrimitive(model)) {
+    let existingModelAlias = importCollection.getMappedAlias(ImportSource.LOCAL_MODELS, model);
+    if (!existingModelAlias) {
+      existingModelAlias = `${model}Aliased`;
```

Comment:
prefer to suffix with `Model` as this is more in line with what a customer would write if found in this situation

Reply:
I was leaning towards a suffix that is less likely to collide again

Reply:
How can you be certain that a model doesn't already collide when adding `Alias` to it?

Reply:
Yeah that is true as well. Ill add a short uuid like string to it